app/components/browse/call_number_navigation.rb

```diff
@@ -5,11 +5,11 @@ class CallNumberNavigation < Navigator
     delegate :next_item, :previous_item, to: :list
 
     def previous_path
-      browse_path(ending: previous_item.key, length: length)
```

Comment:
add a helper method

``` ruby
def classification
  params[:classification]
end
```

this becomes:

``` ruby
browse_path(ending: previous_item.key, length: length, classification: classification)

Reply:
done